src/main/CommandHandler.cpp

```diff
@@ -290,6 +290,11 @@ CommandHandler::logRotate(std::string const& params, std::string& retStr)
 void
 CommandHandler::connect(std::string const& params, std::string& retStr)
 {
+    if (mApp.getConfig().RUN_STANDALONE)
```

Comment:
we should just not register (ie `addRoute`) the overlay related endpoints in standalone mode:
connect, droppeer, peers, quorum, scp, surveytopology, stopsurvey, getsruveyresult, unban

Reply:
ok.